vite.config.router.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from 'path';
+import { defineConfig } from 'vite';
+import Vue from '@vitejs/plugin-vue';
+import ViteSvgLoader from 'vite-svg-loader';
+import Visualizer from 'rollup-plugin-visualizer';
+
+export default defineConfig({
+  root: '.',
+  resolve: {
+    alias: {
+      '@': path.join(__dirname, 'src'),
+      '@@': __dirname,
+    },
+  },
+  plugins: [
+    Vue(),
+    ViteSvgLoader(),
+    Visualizer({
+      filename: path.resolve(__dirname, 'report/router-treemap.html'),
```

Comment:
did you open this file ? everything looks OK ?

Reply:
Yes and it was useful! There was a big `lodash` package inside it caused by a bad auto import (`replace` function) in `FButtonLink`.
But it's fixed, everything looks okay by now. :)